### PR TITLE
[TW-1522] Updates for the Manage API Key interface

### PR DESCRIPTION
**Pull request by akinard-postman**
This PR updates the **Manage API Keys** interface documentation around the updates to now include exposed keys in Postman documentation. This primarily updates:
- Screenshots to reflect the UI updates.
- Minor phrasing revisions to include Postman documentation as a scanned resource.

### CODE REVIEW

**Comment by akinard-postman** [comment]
@ronakodhaviya Could you take a look at the updates here and let me know if things look OK to you? You can preview the published version [here](https://tw-1522-manage-api-keys-update-dec.learning.postman-beta.com/docs/administration/managing-your-team/managing-api-keys/). Thanks!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/administration/managing-your-team/managing-api-keys.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ You can set the expiration settings for all API keys that your team creates with
 
 #### Revoking
 
-Enable the **Auto revoke exposed Postman API keys** setting to allow Postman to automatically revoke any publicly exposed API keys found in publicly accessible Postman resources, [GitHub repositories](/docs/administration/managing-your-team/secret-scanner/#protect-postman-api-keys-in-github), and [Gitlab repositories](/docs/administration/token-scanner/#protect-postman-api-keys-in-gitlab). When the Postman Secret Scanner detects any exposed keys in public GitHub or GitLab repositories or public Postman workspaces, it revokes the key and notifies the key's owner by email.
+Enable the **Auto revoke exposed Postman API keys** setting to allow Postman to automatically revoke any publicly exposed API keys found in publicly accessible Postman resources or documentation, [GitHub repositories](/docs/administration/managing-your-team/secret-scanner/#protect-postman-api-keys-in-github), and [Gitlab repositories](/docs/administration/token-scanner/#protect-postman-api-keys-in-gitlab). When the Postman Secret Scanner detects any exposed keys in public GitHub or GitLab repositories or public Postman resources, it revokes the key and notifies the key's owner by email.
```
Comment (ronakodhaviya) [inline src/pages/docs/administration/managing-your-team/managing-api-keys.md:70]: @akinard-postman 

> "...found in publicly accessible Postman resources or documentation,..."

documentation is part of the publicly accessible Postman resource, we can only say 

> "...found in publicly accessible Postman resources,..."

or specify public workspace and public documentation.

Comment (akinard-postman) [reply src/pages/docs/administration/managing-your-team/managing-api-keys.md]: Thanks @ronakodhaviya. I've reverted the change that was made here.

**Review by mark-anthony-postman — APPROVED** [review]
A few links need fixing, otherwise LGTM!

File: src/pages/docs/administration/managing-your-team/managing-api-keys.md [hunk 2]
```diff
@@ -39,17 +39,17 @@ Tags next to the API key's name specify when it is disabled or revoked. You can
...
 
-If the [Postman Secret Scanner](https://learning.postman.com/docs/administration/token-scanner/) detects exposed Postman API keys in public Postman workspaces or [GitHub](/docs/administration/token-scanner/#protecting-postman-api-keys-in-github) and [GitLab](/docs/administration/token-scanner/#protecting-postman-api-keys-in-gitlab) repositories, the dashboard displays them in the **Exposed API keys** section. This section provides details about the exposed key, such as the date of its exposure detection, the last time it was used, and the location of the exposed API key.
+If the [Postman Secret Scanner](https://learning.postman.com/docs/administration/token-scanner/) detects exposed Postman API keys in public Postman workspaces, public Postman documentation, or [GitHub](/docs/administration/token-scanner/#protecting-postman-api-keys-in-github) and [GitLab](/docs/administration/token-scanner/#protecting-postman-api-keys-in-gitlab) repositories, the dashboard displays them in the **Exposed API keys** section. This section provides details about the exposed key, such as the date of its exposure detection, the last time it was used, and the location of the exposed API key.
```
[the remark this answers is not in the document]
Comment (akinard-postman) [reply src/pages/docs/administration/managing-your-team/managing-api-keys.md:42]: Thanks for catching those links! 🙇

**Comment by akinard-postman** [comment]
Thanks @mark-anthony-postman for checking/catching those links. I thought I'd checked them. 😅

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Pull request merged**